onnxruntime/test/perftest/ort_test_session.cc

```diff
@@ -73,7 +85,12 @@ OnnxRuntimeTestSession::OnnxRuntimeTestSession(Ort::Env& env, std::random_device
     std::string device_id = ""; // [enable_vpu_fast_compile]: Fast-compile may be optionally enabled to speeds up the model's compilation to VPU device specific format.
     size_t num_of_threads = 8; // [num_of_threads]: Overrides the accelerator default value of number of threads with this value at runtime.
 
-    std::istringstream ss(performance_test_config.run_config.ep_runtime_config_string);
+    #ifdef _MSC_VER
+    std::string ov_string = utf8_encode(performance_test_config.run_config.ep_runtime_config_string);
```

Comment:
can we just use ToMBString() in common.h ?
see https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/core/common/str_helper.cc#L9

Reply:
Done